internal/internal_worker.go

```diff
@@ -304,7 +309,8 @@ func newWorkflowTaskWorkerInternal(
 		taskWorker:        localActivityTaskPoller,
 		identity:          params.Identity,
 		workerType:        "LocalActivityWorker",
-		shutdownTimeout:   params.WorkerStopTimeout},
+		shutdownTimeout:   params.WorkerStopTimeout,
+		statsCollector:    params.StatsCollector},
```

Comment:
nit: let's move the closing curly bracket to new line to better indent the contents of `baseWorkerOptions`